uwblueprint/childrens-aid-society
# PUT Case implementation + more Cases Context

## Notion ticket link

[Link and Integrate PUT Case Submission Pt. 2](https://www.notion.so/uwblueprintexecs/134e1f159caa4f73939d11a700c325ad?v=39cf7d5a98384eb9af2bff2265d0ac8a&p=820d94c0c523427ca6fae75724ee2831&pm=s)



## Implementation description
* implemented put case submission API call integration for cases with status: pending, submitted, archived
* refactored some parts of code to use IntakeValueContext and CasesContext



## Steps to test
1. click on a case with pending, submitted or archived status
2. view dynamic values on there and see if it matches with entry in db
3. click review and edit any of the values
4. changes should reflect on both frontend and backend



## What should reviewers focus on?
* family name can't be changed for some reason - I can look into this in another ticket (I can make the bug ticket)
* backend tests are failing in main - I can also look into this in another ticket if you want
* we can sync up on this in a work session or call just lmk


## Checklist
